common/cards/hermits/solidaritygaming-rare.ts

```diff
@@ -0,0 +1,99 @@
+import {HERMIT_CARDS} from '..'
+import {CardPosModel} from '../../models/card-pos-model'
+import {GameModel} from '../../models/game-model'
+import {RowPos} from '../../types/cards'
+import {RowStateWithHermit} from '../../types/game-state'
+import {getNonEmptyRows} from '../../utils/board'
+import HermitCard from '../base/hermit-card'
+import { applyAilment, removeAilment } from '../../utils/board'
+
+class SolidaritygamingRareHermitCard extends HermitCard {
+	constructor() {
+		super({
+			id: 'solidaritygaming_rare',
+			numericId: 156,
+			name: 'Jimmy',
+			rarity: 'rare',
+			hermitType: 'prankster',
+			health: 270,
+			primary: {
+				name: 'The Law',
+				cost: ['prankster', 'any'],
+				damage: 30,
+				power: 'Choose one of your AFK Hermits to protect. This Hermit does not take damage on their first active turn. Only one Hermit can be protected.',
+			},
+			secondary: {
```

Comment:
This move is underpowered for what it costs

Reply:
Gonna class this as resolved because it's temporary